jdaviz/core/template_mixin.py

```diff
@@ -2120,39 +2150,57 @@ def _get_mark_coords(self, viewer):
             else:
                 wcs = getattr(viewer.state.reference_data, 'coords', None)
                 if wcs is None:
-                    return [], []
+                    validity = {'is_aperture': False,
+                                'aperture_message': 'invalid wcs'}
+                    return [], [], validity
                 pixel_region = spatial_region.to_pixel(wcs)
             roi = regions2roi(pixel_region)
 
             # NOTE: this assumes that we'll apply the same radius factor to all subsets (all will
             # be defined at the same slice for cones in cubes)
+#            if self.scale_factor == 1.0:  # this would catch annulus, which might cause confusion
+#                pass
```

Comment:
Can this be removed?

Reply:
I could rewrite it as a comment, but do think we want something here so that we don't accidentally reintroduce it (since it _feels_ like this should be there)